service/src/main/kotlin/io/provenance/explorer/domain/models/explorer/CommonModels.kt

```diff
@@ -38,3 +38,14 @@ data class TokenSupply(
     val communityPool: CoinStr,
     val bonded: CoinStr
 )
+
+data class TokenDistributionAmount(
+    val denom: String,
+    val amount: String
+)
+
+data class TokenDistribution(
+    val holder: String,
```

Comment:
This isnt actually the `holder`, this is the `range`, is that right?

Reply:
Yes. that is `range`. I was playing around with `holder`, `top_holders` and `range`. I'll change it to range.